src/timedate/timedated.c

```diff
@@ -467,6 +469,8 @@ static int match_job_removed(sd_bus_message *m, void *userdata, sd_bus_error *er
                 (void) sd_bus_emit_properties_changed(sd_bus_message_get_bus(m),
                                                       "/org/freedesktop/timedate1", "org.freedesktop.timedate1", "NTP",
                                                       NULL);
+                (void) sd_bus_reply_method_return(c->ntp_req, NULL);
+                c->ntp_req = sd_bus_message_unref(c->ntp_req);
```

Comment:
i'd condition these on the field being set. i mean, it should be set right now always, but it still fields weird to just rely on it.

(assert would work too, but I'd just conditionalize it with if too)